### ISSUE TITLE
Request Queue Size Unbounded

### ISSUE THREAD

**Issue by ajspeck** [plotly/dash-renderer#27]
The changes in #22 cause the request queue to continuously grow as new updates come in since old requests are only marked as rejected and not removed from the list.  Once this array reaches a couple of hundred items long, I see Chrome and Firefox slow to a crawl.  Note the growth of the array is easily visible in the SET_REQUEST_QUEUE action logged in the browser console.  For my app this occurs in a couple of seconds given the number of updates being made.

Adding this:
```
                    dispatch(setRequestQueue(
                        reject(
                            rejected => rejected,
                            getState().requestQueue	
                        )
                    )); 
```
to the updateRequestQueue function 
https://github.com/plotly/dash-renderer/blob/401b97742f675aa16b3e62ace63718ff232d8def/src/actions/index.js#L397-L411
fixed the performance issue I saw but it is not clear that this is the best solution.

**Comment by chriddyp** [comment]
Thanks for reporting! I'll look into this today.

**Comment by chriddyp** [comment]
@ajspeck - I've pushed up a preliminary fix in the prerelease channel. Could you try it out with:
```
pip install dash-renderer==0.12.0rc2
```

This will fix the boundless `requestQueue` issue. You can examine the `requestQueue` yourself by opening up your browser's console and typing `store.getState().requestQueue`. I'm curious to see if this fixes your performance issues.

Thanks!!

**Comment by ajspeck** [comment]
The updated version fixes the performance issues I saw before.  Watching the requestQueue size over 30 minutes, I saw it stayed fixed at around 40 items with none of the growth I saw before.  Thanks for getting this fixed quickly.

**Comment by chriddyp** [comment]
Thank you @ajspeck ! I'll work on getting this into the stable branch soon. In the meantime, it should be "safe" for you to use this prerelease channel.

**Comment by chriddyp** [comment]
Fixed in #28  and available in a stable channel at `dash-renderer==0.11.1`
